apps/www/components/Questionnaire/QuestionHeader.js

```diff
@@ -0,0 +1,23 @@
+import questionStyles from './questionStyles'
+import { Editorial } from '@project-r/styleguide'
+
+const QuestionHeader = ({ metadata }) => {
+  const questionHeader = metadata && metadata.questionHeader
+  if (!questionHeader) {
+    return null
+  }
+  return (
+    <div {...questionStyles.subheadOuter}>
+      <Editorial.Subhead {...questionStyles.subhead}>
+        <span
+          {...questionStyles.subheadInner}
+          style={{ backgroundColor: '#D9F0EF' }}
```

Comment:
Is this intended to stay?

<img width="754" alt="image" src="https://github.com/user-attachments/assets/628e58b4-406e-4785-ab4b-24a9180d00e7" />

Reply:
this is was a suggestion from vivienne and is in discussions with silvia...

Reply:
Yes, I'm not against it but the padding at the very beginning looks a bit odd, and we might want to reuse an existing color?